lib/cli/ui/progress.rb

```diff
@@ -72,7 +72,7 @@ def tick(percent: 0.01, set_percent: nil)
       # Format the progress bar to be printed to terminal
       #
       def to_s
-        suffix = " #{(@percent_done * 100).round(2)}%"
+        suffix = sprintf(" %.2f%", @percent_done * 100)
```

Comment:
If `@percent_done == 1`, I think it’d be nicer to print a neat `100%`.
```
1.00%
10.00%
100%
```

It kinda ties everything together at the end.

Reply:
I agree with @lavoiesl, but there are also many cases where all intermediate values will be integer percentages, and it would be nice to show without decimals in that case.  What about 
```suggestion
        suffix = sprintf(" %5s%", (percent_done * 100).round(2).to_s.chomp('.0'))
```
5 gives enough space for all of the following - 3 digits will never be followed by decimal places.
```
    1
  1.1
 1.11
   11
 11.1
11.11
  100
```

Reply:
That’s even better 🎉